### ISSUE TITLE
ActiveRecord rawSql strange bug

### ISSUE THREAD

**Issue by nicdnepr** [yiisoft/yii2-sphinx#87]
### What steps will reproduce the problem?
```php
class Sticker extends yii\sphinx\ActiveRecord
public static function indexName()
    {
        return 'sticker';
    }
...
$q=Sticker::find();
$q->andWhere(['>=', 'pricecsofferme', 5]);
var_dump($q->createCommand()->rawSql);
```
 
### What's expected?
```SELECT * FROM `sticker` WHERE `pricecsofferme` >= 5```
### What do you get instead?
```SELECT * FROM `sticker` WHERE `pricecsofferme` >= :qp0```

### Additional info

| Q                     | A
| --------------------- | ---
| Yii version           |2.0.12
| Yii Sphinx version    |2.0.8
| Sphinx server version |Sphinx 2.2.11-id64-release (95ae9a6)
| PHP version           |7
| Operating system      |ubuntu 16

**Comment by samdark** [comment]
It's not a bug. We're using prepared statements and `:qp0` is a placeholder.

**Comment by nicdnepr** [comment]
If replace 'sticker' to any word like 'sticker0' or 'test' it works fine and return ```pricecsofferme >= 5```

**Comment by nicdnepr** [comment]
The reason for this question is this code
```
$sticks = static::getDb()->cache(function () use ($query) {
       return $query->asArray()->all();
}, Yii::$app->params['cache']);
```
because of rawSql is the same for any params, it returns same data for any params

**Comment by cebe** [comment]
> var_dump($q->createCommand()->rawSql);

rawSql should have 5 instead of :qp0

**Comment by klimov-paul** [comment]
Resolved by commit aa6ec830e6abfe68e4e4f5bdc581c8d2b891dabc
